Assignment2/jhanderunk-JhosefCardich.ttf

```diff
@@ -0,0 +1,14 @@
+@base rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix example: <http://www.ontologies.org/ontologies/assignment2#>
+
+:Class01 example:includes :Sensor029;
+		 example:includes :Computer101.
+
+:User10A example:hasName :"Pedro"^^xsd:string;
```

Comment:
Wrong with : before the data. Same below

Reply:
: removed .